tests/CMakeLists.txt

```diff
@@ -27,9 +27,20 @@ if(REALM_ENABLE_SYNC)
     list(APPEND SOURCES sync.cpp)
 endif()
 
+find_package(Threads)
+find_package(PkgConfig REQUIRED)
+pkg_search_module(OPENSSL REQUIRED openssl)
```

Comment:
Why is this adding a hard dependency on OpenSSL here? The tests don't use OpenSSL directly on any platform. If the existing logic for making OpenSSL a dependency of the objectstore library is incorrect then that should be updated instead.